### DISCUSSION TITLE
Do you have any plan for linux client?

### DISCUSSION THREAD

**Discussion by asoMansoury** [vpnhood/VpnHood#302] [General]
I don't know why you haven't released any client for linux clients?

**Comment by trudyhood** [comment]
Duplicate of #45 
We just don't have time to implement, deploy and test it this time. Can you do that?

**Reply by asoMansoury** [reply]
I'm OK. 
Please give me documentation to set up the project and start developing.
I'm really excited about developing your project. By which Language I should start?

**Reply by trudyhood** [reply]
> I'm OK. Please give me documentation to set up the project and start developing. I'm really excited about developing your project. By which Language I should start?

Thats's great! It is C#, and you don't need to do much work.
Just implement this interface:
https://github.com/vpnhood/VpnHood/blob/main/VpnHood.Client.Device/IPacketCapture.cs

Like how we did for Windows:
https://github.com/vpnhood/VpnHood/tree/main/VpnHood.Client.Device.WinDivert

You can use the same library that we used. I think they support Linux too:
https://github.com/dotpcap/sharppcap

I am here for you if you need help.
